go/buildhelper/client.go

```diff
@@ -0,0 +1,96 @@
+package buildhelper
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/go/ethclient"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+)
+
+type BuildHelperClient struct {
+	node *EthNode
+}
+
+func (b *BuildHelperClient) FetchBlockByNumber(n *big.Int) (*types.Block, error) {
```

Comment:
@otherview Can we kill this off, here and in the interface? Doesn't seem to be used at all. Or are we keeping it around for something?

Reply:
That's a great catch, I'm not sure why it came to be if it's not being used 🤔  Thanks!